packages/react/fast-components-react-base/.prettierignore

```diff
@@ -1,3 +1,4 @@
 coverage/*
 dist/*
-www/*
\ No newline at end of file
+www/*
+src/**/.tmp
```

Comment:
`.tmp` is a dir - does it need a `/`?

Reply:
It seemed to require this because it's buried inside component folders

Reply:
I think hes saying is should these be `src/**/.tmp/` because for folders we include a trailing slash.

Reply:
Thanks for the clarification Jane.